tests/README.md

```diff
@@ -0,0 +1,78 @@
+## Testing dbt project: `jaffle_shop`
```

Comment:
I found it a bit confusing that the sample dbt project is directly in the `tests` root folder. I would find it a bit more tidy if it were on a folder like `fixtures/sample_project` or something similar.

Reply:
Agreed, moved to a fixtures folder in the test dir